### PR TITLE
Remove unnecessarily escaped characters.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackagePartName.java [hunk 1]
```diff
@@ -326,7 +326,7 @@ private static void checkPCharCompliance(String segment)
             /* Check rule M1.7 */
             if (decodedChar == '/' || decodedChar == '\\') {
                 throw new InvalidFormatException(
-                    "A segment shall not contain percent-encoded forward slash ('/'), or backward slash ('\') characters. [M1.7]");
+                        "A segment shall not contain percent-encoded forward slash ('/'), or backward slash ('') characters. [M1.7]");
```
Comment (pjfanning) [inline poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackagePartName.java]: this change is wrong

Comment (pjfanning) [reply poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackagePartName.java]: or at least - I think the aim is to produce the text `'\'` - the fix might be to use `'\\'`

Comment (arturobernalg) [reply poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackagePartName.java]: reverted.

**Comment by pjfanning** [comment]
@arturobernalg we don't use git - we use svn. Usually, if the Github PR is worthwhile, I'll go through the effort of creating a svn commit. In this case, I don't think the new code is a big improvement. I'm going to opt out of this. Maybe some other POI dev will like the change and apply it.

**Pull request closed without merging**